### PR TITLE
Template - Replication

### PR DESCRIPTION
**Pull request by Or-Gabay**

### CODE REVIEW

File: artifactory/commands/replication/template.go [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+		valueQuestion.MapKey = key
+		valueQuestion.PromptPrefix = InsertValuePromptMsg + key
+		if valueQuestion.Options != nil {
+			valueQuestion.PromptPrefix += utils.PressTabMsg
+		}
+		valueQuestion.PromptPrefix += " >"
+		value, err = iq.AskQuestion(valueQuestion)
+	}
+	return value, err
+}
```
Comment (yahavi) [inline artifactory/commands/replication/template.go]: This code repeated in repository/template.go
I suggest moving the function to questionnaire.go

File: artifactory_test.go [hunk 2]
```diff
@@ -3663,3 +3663,34 @@ func initVcsTestDir(t *testing.T) string {
 	assert.NoError(t, err)
 	return path
 }
+func TestArtifactoryReplicationCreate(t *testing.T) {
```
Comment (yahavi) [inline artifactory_test.go]:
```suggestion

func TestArtifactoryReplicationCreate(t *testing.T) {
```

File: artifactory_test.go [hunk 3]
```diff
@@ -3663,3 +3663,34 @@ func initVcsTestDir(t *testing.T) string {
...
+	initArtifactoryTest(t)
+	configArtifactoryCli.Exec("c", tests.RtServerId, "--url="+*tests.RtUrl, "--user="+*tests.RtUser, "--password="+*tests.RtPassword, "--interactive=false")
+
```
Comment (yahavi) [inline artifactory_test.go]: I sugget moving deleteServerConfig here:
```
defer deleteServerConfig()
```

File: artifactory_test.go [hunk 4]
```diff
@@ -3663,3 +3663,34 @@ func initVcsTestDir(t *testing.T) string {
 	assert.NoError(t, err)
 	return path
 }
+func TestArtifactoryReplicationCreate(t *testing.T) {
+	initArtifactoryTest(t)
+	configArtifactoryCli.Exec("c", tests.RtServerId, "--url="+*tests.RtUrl, "--user="+*tests.RtUser, "--password="+*tests.RtPassword, "--interactive=false")
```
Comment (yahavi) [inline artifactory_test.go]: This test will not work with ApiKey and AccessToken. I suggest setting them too

File: docs/artifactory/replicationtemplate/help.go [hunk 5]
```diff
@@ -0,0 +1,8 @@
+package replicationtemplate
+
+const Description = "Create a JSON template for creation replication repository."
+
+var Usage = []string{`jfrog rt rplt <template path>`}
+
+const Arguments string = `	template path
+		Specifies the local file system path for the template file to be used for the replication create. The template can be created using the “jfrog rt rplt” command.`
```
[the remark this answers is not in the document]
Comment (Or-Gabay) [reply docs/artifactory/replicationtemplate/help.go]: Eyal added this

[the remark this answers is not in the document]
Comment (yahavi) [reply docs/artifactory/replicationtemplate/help.go]: Still it doesn't have to be in the template command

**Pull request merged**